### PR TITLE
Since the snapshot are correct and are available we can turn on the integration tests.

### PR DESCRIPTION
**Pull request by ph**
Since the snapshot are correct and are available we can turn on the integration tests. 

need to be ported to 6.6,7.x, when the snapshots are available.

master snapshots are not yet available.

### CODE REVIEW

**Comment by ph** [comment]
jenkins test this

**Comment by ph** [comment]
@urso lets get that merged in 7.0, snapshots are available in 8.0 I'll forward port it.

**Comment by ph** [comment]
rebased for filebeat failures.

File: x-pack/libbeat/tests/system/test_management.py [hunk 1]
```diff
@@ -25,10 +27,13 @@ def setUp(self):
         self.es_user = "myelastic"
         self.es_pass = "changeme"
         self.es = Elasticsearch([self.get_elasticsearch_url()], verify_certs=True)
+        self.keystore_path = self.working_dir + "/data/keystore"
```
Comment (urso) [inline x-pack/libbeat/tests/system/test_management.py]: In general, better use `os.path.join`. We should also consider to 'abstract' some functionality around beats into classes.

File: x-pack/libbeat/tests/system/test_management.py [hunk 2]
```diff
@@ -231,10 +226,9 @@ def check_kibana_status(self):
 
         r = requests.get(url, headers=headers,
                          auth=(self.es_user, self.es_pass))
-        print(r.text)
 
     def create_and_assing_tag(self, blocks):
-        tag_name = "test"
+        tag_name = "test%d" % int(time.time() * 1000)
```
Comment (urso) [inline x-pack/libbeat/tests/system/test_management.py]: nit: `"...".format(...)`

File: x-pack/libbeat/tests/system/test_management.py (hunk 1 above)
Comment (ph) [reply x-pack/libbeat/tests/system/test_management.py]: I will make that change, I can create an issue for abstract.

**Pull request merged**